engines/bops_api/app/services/bops_api/application/creation_service.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+module BopsApi
+  module Application
+    class CreationService
+      class CreateError < StandardError; end
+
+      def initialize(**options)
+        raise CreateError, "Creating planning applications using this endpoint is not permitted in production" if Bops.env.production?
```

Comment:
Seems like this is a different category of error to one where we capture an active record error?